bin/i18n/i18n_script_utils.rb

```diff
@@ -335,32 +340,28 @@ def self.sort_and_sanitize(hash)
     end
   end
 
+  def self.json_file?(file_path)
+    %w[.json].include? File.extname(file_path)
```

Comment:
Should we consider case-sensitivity?
```suggestion
    %w[.json].include?(File.extname(file_path).downcase)
```

Reply:
I haven't found a single file on the project where extensions `.json`, `.yml`, or `.yaml` were in the upper case, but it wouldn't be superfluous to play it safe just in case